packages/react-html/src/manager.ts

```diff
@@ -41,7 +42,7 @@ export class HtmlManager {
 
     return {
       title: lastTitle && lastTitle.title,
-      metas: this.metas,
+      metas: uniqby(uniqby(this.metas.reverse(), 'name'), 'property'),
```

Comment:
I am opened to suggestions if we can improve the logic here.
I am reversing the array to make sure we keep the latest entry in the array.

Reply:
I made a [codesandbox](https://codesandbox.io/s/unruffled-moon-hihfq?file=/src/index.js) to illustrate the logic here.